tooling/tracer/src/tracer_glue.rs

```diff
@@ -85,7 +83,21 @@ pub(crate) fn register_call(tracer: &mut Tracer, location: &SourceLocation, fram
     let path = &PathBuf::from(filepath.to_string());
     let line = Line(*line_number as i64);
     let file_id = tracer.ensure_function_id(&frame.function_name, path, line);
-    tracer.register_call(file_id, vec![]);
+    let args = convert_params_to_args_vec(tracer, frame);
+    tracer.register_call(file_id, args);
+}
+
+/// Extracts the relevant information from the given `frame` to construct a vector of `ArgRecord`
+/// that the `Tracer` interface expects when registering function calls.
+fn convert_params_to_args_vec(tracer: &mut Tracer, frame: &StackFrame) -> Vec<ArgRecord> {
+    let mut result = Vec::new();
+    for param_index in &frame.function_param_indexes {
+        let variable = &frame.variables[*param_index];
+        // TODO(stanm): maybe don't duplicate values?
+        let value_record = register_value(tracer, &variable.value, &variable.typ);
+        result.push(ArgRecord { name: String::from(&variable.name), value: value_record });
```

Comment:
forgive me, I could warn you, but I delayed that: we updated to `0.5.9`, and we don't support `ArgRecord` anymore: we use similar structures and  mechanisms that we use for the other local variables now: so one should just do `tracer.arg(&variable.name, value_record)` and add that to the `result`

Reply:
Thanks for the note. I will update to 0.5.9 and will change this code before merging.

Reply:
Done.